AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/helper/ModelConstants.java

```diff
@@ -49,4 +51,12 @@
 	private ModelConstants() {
 	}
 
+	public static void init(ProcessingEnvironment processingEnv) {
+		OptionsHelper optionsHelper = new OptionsHelper(processingEnv);
```

Comment:
and what SourceVersion value to use? `SourceVersion.RELEASE_6` or `SourceVersion.RELEASE_7`?